static/js/containers/pages/login/LoginForgotPasswordPage.js

```diff
@@ -19,34 +20,47 @@ import type { EmailFormValues } from "../../../components/forms/EmailForm"
 
 type Props = {
   history: RouterHistory,
-  forgotPassword: (email: string) => Promise<any>,
-  addUserNotification: Function
+  forgotPassword: (email: string) => Promise<any>
+}
+
+type State = {
+  forgotEmailSent: boolean,
+  text: string
```

Comment:
I could be wrong but for consistency, can we add new state https://github.com/mitodl/mitxpro/blob/master/static/js/lib/auth.js#L48 instead of managing local state as we are using redux to store globe state at one place.

Reply:
@tasawernawaz , I am not sure but actually, using that state here is at the component level but not global store and It seems me, no any other component(s) will utilize this state. currently, declaring it as redux  does not mean to me.